style.css

```diff
@@ -41,3 +41,180 @@
 .green {
   color: #36b37e;
 }
+
+.line {
+  width: 48px;
+  height: 4px;
+  background-color: #36b37e;
+}
+
+.work-section {
+  width: 100%;
+  align-items: cente;
+  padding: 10px 15px 15px 10px;
+}
+
+.project-container {
+  display: grid;
+  grid-template-columns: 80%;
+  grid-template-rows: auto auto auto auto auto auto;
+  gap: 20px;
+}
+
+.project {
+  border: 1px solid green;
+  margin: 0;
+  width: 100%;
+  padding: 0;
+  border-radius: 10px;
+}
```

Comment:
The content of the individual project should be aligned with flexbox as stated in the project requirements (check the bullet point number 2 on the image below).

Also, kindly try as much as possible to stick to the figma design. For example, each individual project card has a width and height of 327px and 474px respectively, but your card has width and height of 374.39px and 414px respectively. 

![Screenshot from 2022-09-23 12-45-52](https://user-images.githubusercontent.com/65233860/191966128-ad53a763-77e7-4f25-a598-cf45635a64a5.png)